scipy/stats/_continuous_distns.py

```diff
@@ -5953,20 +5953,23 @@ class pearson3_gen(rv_continuous):
 
     .. math::
 
-        f(x, skew) = \frac{|\beta|}{\Gamma(\alpha)}
-                     (\beta (x - \zeta))^{\alpha - 1}
-                     \exp(-\beta (x - \zeta))
+        f(x, \kappa) = \frac{|\beta|}{\Gamma(\alpha)}
+                       (\beta (x - \zeta))^{\alpha - 1}
+                       \exp(-\beta (x - \zeta))
```

Comment:
Skew renders as four italicized letters, which can be confused with the product of s, k, e, and w
We could make put it in an \mbox, but I think \kappa is more readable in plaintext and when rendered.